src/ontology/uberon-edit.obo

```diff
@@ -43863,6 +43862,8 @@ xref: SCTID:278981008
 xref: UMLS:C0225666 {source="ncithesaurus:Terminal_Bronchiole"}
 xref: Wikipedia:Terminal_bronchiole
 is_a: UBERON:0002186 ! bronchiole
+relationship: distally_connected_to UBERON:0002188 ! respiratory bronchiole
+relationship: distally_connected_to UBERON:0008874 ! pulmonary acinus
```

Comment:
The text def of connected to:
"a is connected to b if and only if a and b are discrete structure, and there exists some connecting structure c, such that c connects a and b"

In this relationship, what is 'c'? Can it be 'respiratory bronchiole' if 'respiratory bronchiole' is already asserted to be part of the 'pulmonary acinus'?

Reply:
yes, in this case 'respiratory bronchiole' would be c and the usage of connected_to would be more aligned with the definition.